ChangeLog

```diff
@@ -1,36 +1,76 @@
 [TOC]
 
-## post-1.0 until [git 5b6e9ba63d4b083015f5c8f7360b96e38f607a04](https://github.com/widelands/widelands/commit/5b6e9ba63d4b083015f5c8f7360b96e38f607a04)
+## post-1.0 until [git 7c0a3e7](https://github.com/widelands/widelands/commit/7c0a3e7242e37ca74696d75f0f72db7db678c151)
 
 ### Animations, Icons and Overlays
 
 - Fix some "deprecated pictures parameter" warnings (#4908)
 - Create spritesheets (#4935)
 - Spritesheet cleanup & campaigns (#4974)
 - Updated barbarian barracks graphics by DragonAtma (#4977)
+- Spritesheets fixes and cleanups (#5049)
 
 
 ### Sounds and Music
 
 - Write config after sound initialization (#4940)
+- Enable custom in-game music (#5138)
 
 
 ### Tutorials and Campaigns
 
-
+- Emp02: Allow sheep farm and weaving mill with barracks (#5048)
+- Add "Along the river" scenario (#5062)
 
 
 ### Scripting
 
 - Replenish reed in Frisian trading outpost (#4955)
 - Add enhancement method for buildings to Lua (#4989)
+- Added more ship names for Amazon tribes (#5034)
+- Added lore for Dressmakery (#5037)
+  - Added lore for Dressmakery
```

Comment:
Same text in two lines.

Otherwise LGTM :-)

Thanks for keeping this up to date :1st_place_medal: